roadmap.md

```diff
@@ -0,0 +1,87 @@
+.NET Core Roadmap
+=================
+
+.NET Core is a general purpose, modular, cross-platform and open source implementation of .NET. The .NET Core team (anyone contributing to .NET Core) aspires to enable it to work for many interesting scenarios. Some of those are already part-way to completion. Nice work, team! 
+
+This roadmap is intended to communicate the team's priorities for evolving and extending the scope of .NET Core.
+
+Microsoft is very invested in .NET Core and makes multiple "official Microsoft" .NET Core distributions available to its customers. Sweet! It is expected (and desired) that other groups (corporations or otherwise) will also become significantly invested in .NET Core, possibly making other distros available. As a result, this roadmap is written from a neutral perspective, not a Microsoft one (unless specifically called out). This document is therefore naturally open to PRs.
+
+Project Goals
+=============
+
+Broad goals:
+
+- .NET Core code is high quality (important consideration for contributions).
+- .NET Core can be ported to a broad set of OS platforms and chip architectures.
+- .NET Core has compelling performance and is highly reliable.
+- Developers can acquire a .NET Core developer environment quickly and intuitively.
+- Developers can productively and intuitively build apps, using documentation, samples and NuGet components.
+
+Active ports:
+
+- OSes: Linux, OS X, FreeBSD, Rasberry Pi2 (Linux) and Windows.
```

Comment:
That's fair. We should probably rename that section to "Platforms: ". Strictly speaking, Windows isn't a singular platform either, there is Win32 and AppContainer, for example.